### PR TITLE
add examples from readme

### PR DESCRIPTION
**Pull request by ralphtheninja**
I extracted some of the code from the `README` so you can run examples quick rather than having to copy paste etc.

### CODE REVIEW

**Comment by medikoo** [comment]
Thanks, I'm happy to take. Can you merge it with master and assure that it passes Travis build?

**Comment by ralphtheninja** [comment]
Sure but what's going on here? These are just example scripts. Why would they have anything to do with travis?

**Comment by medikoo** [comment]
@ralphtheninja there's a linter bound to travis, and all files in a project should confirm to defined lint rules (eventual errors that can be picked via static analysis, whitespace conventions etc)

**Comment by ralphtheninja** [comment]
Ok. I first thought that tad tried running corresponding test files from the examples folder. Let me try again.

**Comment by medikoo** [comment]
@ralphtheninja yes, that was the case, but that should be fixed once you assure you're up to date with master

**Comment by medikoo** [comment]
.. and be sure to reinstall dependencies after update

**Comment by ralphtheninja** [comment]
Done.

**Comment by ralphtheninja** [comment]
Oh, I have to run the linter locally. Sorry, adjusting :)

**Comment by medikoo** [comment]
Ok :) You can try: "npm run lint-console", you'll have live overview

**Comment by ralphtheninja** [comment]
Got most fixed but have problems finding out what `Unexpected '(space)'` means:

unexpected-space

**Comment by ralphtheninja** [comment]
Cool feature btw `npm run lint-console`!

**Comment by medikoo** [comment]
@ralphtheninja it's probably because indentation is done with _spaces_ while it should be with _tabs_

**Comment by ralphtheninja** [comment]
There we go.

File: examples/throbber.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+var throbber = setupThrobber(function (str) {
+	process.stdout.write(str);
+}, 200);
+
+process.stdout.write('Throbbing for 3 seconds here -> ');
+throbber.start();
+
+setTimeout(function () {
+	console.log();
```
Comment (medikoo) [inline examples/throbber.js]: Is there any purpose of that log here?

Comment (medikoo) [inline examples/throbber.js]: I see that it's probably for a new line

**Comment by medikoo** [comment]
Thank you!

File: examples/throbber.js (hunk 1 above)
Comment (ralphtheninja) [inline examples/throbber.js]: Aye, to get the prompt on a new line afterwards.
